pyproject.toml

```diff
@@ -31,76 +31,85 @@ classifiers = [
     "License :: OSI Approved :: MIT License",
 ]
 
+requires-python = ">=3.9"
+
 # NOTE: Minimum dependencies are tested with the packages defined in the
 # `ci/environment-mindeps.yaml` conda environment. When updating minimum dependencies
 # here, make sure to also update `ci/environment-mindeps.yaml`.
-[tool.poetry.dependencies]
-python = ">=3.9,<4.0"
-python-cmr = ">=0.10.0"
-pqdm = ">=0.1"
-requests = ">=2.26"
-s3fs = ">=2022.11"
-fsspec = ">=2022.11"
-tinynetrc = "^1.3.1"
-multimethod = ">=1.8"
-kerchunk = { version = ">=0.1.2", optional = true }
+dependencies = [
+  "python-cmr >=0.10.0",
+  "pqdm >=0.1",
+  "requests >=2.26",
+  "s3fs >=2022.11",
+  "fsspec >=2022.11",
+  "tinynetrc ^1.3.1",
```

Comment:
Caret requirements are not supported in the project.dependencies table.

Reply:
Oh, nevermind. this is still poetry dependencies.

Reply:
Good catch! IMO we shouldn't be using caret notation since it's non-standard.

Reply:
edd6442